db/data/schedules/schedules.csv

```diff
@@ -236,4 +236,31 @@ npq_specialist,npq-specialist-autumn,NPQ Specialist Autumn,2023,Output 2 - Reten
 npq_specialist,npq-specialist-spring,NPQ Specialist Spring,2023,Output 1 - Participant Start,started,01/01/2024,,01/01/2024
 npq_specialist,npq-specialist-spring,NPQ Specialist Spring,2023,Output 2 - Retention Point 1,retained-1,01/01/2024,,01/01/2024
-npq_specialist,npq-specialist-spring,NPQ Specialist Spring,2023,Output 3 - Participant Completion,completed,01/01/2024,,01/01/2024
\ No newline at end of file
+npq_specialist,npq-specialist-spring,NPQ Specialist Spring,2023,Output 3 - Participant Completion,completed,01/01/2024,,01/01/2024
+ecf_extended,ecf-extended-september,ECF Extended September,2021,Output 7 - Extended Point 1,extended-1,01/09/2021,,01/09/2021
+ecf_extended,ecf-extended-september,ECF Extended September,2021,Output 8 - Extended Point 2,extended-2,01/09/2021,,01/09/2021
+ecf_extended,ecf-extended-september,ECF Extended September,2021,Output 9 - Extended Point 3,extended-3,01/09/2021,,01/09/2021
+ecf_extended,ecf-extended-january,ECF Extended January,2021,Output 7 - Extended Point 1,extended-1,01/01/2022,,01/01/2022
+ecf_extended,ecf-extended-january,ECF Extended January,2021,Output 8 - Extended Point 2,extended-2,01/01/2022,,01/01/2022
+ecf_extended,ecf-extended-january,ECF Extended January,2021,Output 9 - Extended Point 3,extended-3,01/01/2022,,01/01/2022
+ecf_extended,ecf-extended-april,ECF Extended April,2021,Output 7 - Extended Point 1,extended-1,01/04/2022,,01/04/2022
+ecf_extended,ecf-extended-april,ECF Extended April,2021,Output 8 - Extended Point 2,extended-2,01/04/2022,,01/04/2022
+ecf_extended,ecf-extended-april,ECF Extended April,2021,Output 9 - Extended Point 3,extended-3,01/04/2022,,01/04/2022
+ecf_extended,ecf-extended-september,ECF Extended September,2022,Output 7 - Extended Point 1,extended-1,01/09/2022,,01/09/2022
+ecf_extended,ecf-extended-september,ECF Extended September,2022,Output 8 - Extended Point 2,extended-2,01/09/2022,,01/09/2022
+ecf_extended,ecf-extended-september,ECF Extended September,2022,Output 9 - Extended Point 3,extended-3,01/09/2022,,01/09/2022
+ecf_extended,ecf-extended-january,ECF Extended January,2022,Output 7 - Extended Point 1,extended-1,01/01/2023,,01/01/2023
+ecf_extended,ecf-extended-january,ECF Extended January,2022,Output 8 - Extended Point 2,extended-2,01/01/2023,,01/01/2023
+ecf_extended,ecf-extended-january,ECF Extended January,2022,Output 9 - Extended Point 3,extended-3,01/01/2023,,01/01/2023
+ecf_extended,ecf-extended-april,ECF Extended April,2022,Output 7 - Extended Point 1,extended-1,01/04/2023,,01/04/2023
+ecf_extended,ecf-extended-april,ECF Extended April,2022,Output 8 - Extended Point 2,extended-2,01/04/2023,,01/04/2023
+ecf_extended,ecf-extended-april,ECF Extended April,2022,Output 9 - Extended Point 3,extended-3,01/04/2023,,01/04/2023
+ecf_extended,ecf-extended-september,ECF Extended September,2023,Output 7 - Extended Point 1,extended-1,01/09/2023,,01/09/2023
```

Comment:
we seem to be missing all the 2023 ECF schedules from this csv, do you mind adding those to the seed data please, as we're only adding the extended now for 7,8,9 only?

Reply:
All done tks @cwrw